packages/earth-admin/src/pages-client/dashboards/routes.tsx

```diff
@@ -49,6 +49,7 @@ export default function DashboardsPage(props) {
     return encodeQueryToURL('dashboards', query);
   };
   const { listProps, mutate } = useInfiniteList(getQuery, getAllDashboards);
+  const selectedItem = props['*'];
```

Comment:
pls leave a comment on what * is, this is very weird to see in the code

Reply:
I could also use `const { id } = useMatch(':id')` to take the id from the url. Could be cleaner